### ISSUE TITLE
Inference graph can use tranformer?

### ISSUE THREAD

**Issue by ChaejinE** [kserve/kserve#3337]
/kind bug

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]

**What did you expect to happen:**
I have a question
I know that predictor can use transformer it is ususally proccessed for predictor's preprocess and postprocess
I want to handle Inference graph like above.
Is it possible using transformer to use preprocess, postprocess? if not, is there anyway?

**What's the InferenceService yaml:**
[To help us debug please run `kubectl get isvc $name -n $namespace -oyaml` and paste the output]

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Environment:**

- Istio Version:
- Knative Version:
- KServe Version:
- Kubeflow version:
- Cloud Environment:[k8s_istio/istio_dex/gcp_basic_auth/gcp_iap/aws/aws_cognito/ibm]
- Minikube/Kind version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

**Comment by sivanantha321** [comment]
@ChaejinE Yes, It is possible to run Inferencegraph with transformer. You have to just create the ISVC with predictor and transformer

**Comment by ChaejinE** [comment]
@sivanantha321 
Ah! thankyou for response first.
but
My question is only inference graph - transformer relation arch, not isvc - transformer set.

for example,
1. transformer preprocess

2. inference graph (isvc1 iscv2...)

3. transformrt postprocess

It needs a transformer for only inference graph, which not related isvc in the inference graph

**Comment by sivanantha321** [comment]
@ChaejinE Currently we don't support adding transformers to the inference graph. Can you elaborate the use case ?

**Comment by yolk-pie-L** [comment]
I use an ensemble node and a switch node. The ensemble node contains three language detectors, which identify the language of the input. The switch node contains several translator, each is reposible for translate one specific language to the other. And the translator is activated when the detected language is the specific language. When I access the ensemble node, its output is like this: 
```
{
  "languages-51": {
    "instances": [
      "de",
      "abcd"
    ]
  },
  "lingua": {
    "instances": [
      "de",
      "abcd"
    ]
  },
  "xlm-roberta-base-language-detec tion": {
    "instances": [
      "es",
      "efgh."
    ]
  }
}
```

So I want to do post processing in the ensemble node. To aggregate the response and give out like
```
    {"instances": [
      "de",
      "abcd"
    ]}
```
Then I can use this data to activate the correspoding translator. I don't know how to this in the current architecture, unless I add a sequnce node to do the processing.
@sivanantha321

**Comment by sivanantha321** [comment]
@rachitchauhan43 Can you help here ?
